### PR TITLE
fw_pos_control_l1: cleanup body velocity and acceleration usage

### PR DESCRIPTION
**Pull request by dagar**

### CODE REVIEW

**Comment by dagar** [comment]
@Antiheavy this should also finally fix the FW hand launch issue when pointing the vehicle up.

File: src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp [hunk 1]
```diff
@@ -303,18 +319,13 @@ FixedwingPositionControl::calculate_target_airspeed(float airspeed_demand, const
...
-		yaw_vector.normalize();
-
-		const float ground_speed_body = yaw_vector * ground_speed;
-
 		/*
 		 * This error value ensures that a plane (as long as its throttle capability is
 		 * not exceeded) travels towards a waypoint (and is not pushed more and more away
 		 * by wind). Not countering this would lead to a fly-away.
 		 */
+		const float ground_speed_body = _body_velocity(0);
```
Comment (dagar) [inline src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp:327]: TODO: review, this isn't right

Comment (RomanBapst) [reply src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp:327]: @dagar What's not right about this? Isn't it the same logic as before?

**Review by RomanBapst — APPROVED** [review]

**Review by sfuhrer — APPROVED** [review]
Nice clean up, didn't see any introduced errors.

File: src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp (hunk 1 above)
Comment (dagar) [reply src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp:327]: The underspeed behaviour when you start climbing/descending.

**Pull request merged**